examples/tools/bevymark.rs

```diff
@@ -1,9 +1,12 @@
+use std::fmt::Write;
```

Comment:
Why have you seperated the groups with newlines?

Reply:
Either my editor or rustfmt did that.

Reply:
Even if so, it's not the correct style; the groups should not have newlines between them. Once you root-cause this, please let us know, as we should complain to the rustfmt folks if this is a regression in the nightly.

Reply:
Looks like this is CLion doing stuff since the default is `Preserve`: https://rust-lang.github.io/rustfmt/?version=master&search=#group_imports. I've learned to just let the tools decide what's right, but I'll smush these imports back together manually.